src/java/org/apache/cassandra/db/streaming/CassandraEntireSSTableStreamWriter.java

```diff
@@ -93,7 +93,7 @@ public void write(StreamingDataOutputPlus out) throws IOException
             long bytesWritten = out.writeFileToChannel(channel, limiter);
             progress += bytesWritten;
 
-            session.progress(sstable.descriptor.filenameFor(component), ProgressInfo.Direction.OUT, bytesWritten, length);
+            session.progress(sstable.descriptor.filenameFor(component), ProgressInfo.Direction.OUT, bytesWritten, bytesWritten, length);
```

Comment:
I think the first usage of `bytesWritten` may be incorrect (everywhere else uses what is calculated as `progress` above). Doesn't have to block this ticket because it's already done this way, but it might be worth looking at in the future.

Reply:
can you explain?  This is whole file streaming, so `bytesWritten` should map to the file length

Reply:
we spoke in Slack, the summary is:

There is the field `progress` that tracks all the bytes written regardless of files, but `session.progress` is relative to the file.  @isaacreath thought that we should do

```
session.progress(sstable.descriptor.filenameFor(component), ProgressInfo.Direction.OUT, progress, bytesWritten, length);
```

This was a misunderstanding and chatting about it cleared things up; we are relative to the file so can't use `progress` in this context